apis/v1alpha2/httproute_types.go

```diff
@@ -478,11 +486,13 @@ const (
 //
 // ```
 // match:
-//   path:
-//     value: "/foo"
-//   headers:
-//   - name: "version"
-//     value "v1"
+//
+//	path:
+//	  value: "/foo"
+//	headers:
+//	- name: "version"
+//	  value "v1"
+//
```

Comment:
IS this change intentional?

Reply:
Looks like this was an issue with my Go toolchain. Fixed.